### ISSUE TITLE
RABBITMQ_CONFIG_FILE may be incorrect

### ISSUE THREAD

**Issue by tashrifbillah** [AMP-SCZ/dpdash#10]
Should the [`RABBITMQ_CONFIG_FILE`](https://github.com/PREDICT-DPACC/dpdash/blob/pnl-devel/singularity/setup.sh#L23) need update? According to [this](https://github.com/PREDICT-DPACC/dpdash/blob/pnl-devel/singularity/configure.py#L68), it is missing `.config` extension.

**Comment by tashrifbillah** [comment]
@blms may have run into an error because of this

**Comment by blms** [comment]
It may be related. The issue I ran into was during the spin-up of a development instance of dpdash on the same machine running a production instance (below, "rc-predict"). Though I was able to specify a different port for RabbitMQ, it seems that it also requires a separate node name. (I thought specifying this might be a simple change, but it seems more complicated than that to [run multiple nodes on one machine](https://www.rabbitmq.com/clustering.html).)

```sh
***************Setting up RABBITMQ***************
Warning: PID file not written; -detached was passed.
ERROR: node with name "rabbit" already running on "rc-predict"
```

But this could be a separate issue.

**Comment by tashrifbillah** [comment]
Solution coming ... (hopefully)

**Comment by tashrifbillah** [comment]
Hard coded node name:
https://github.com/PREDICT-DPACC/dpdash/blob/pnl-devel/singularity/configure.py#L57

Working on it ...

**Comment by tashrifbillah** [comment]
> but it seems more complicated than that to run multiple nodes on one machine

Did you find out why?

**Comment by tashrifbillah** [comment]
The issue has been solved, commits to follow ...

**Comment by blms** [comment]
Closed by #14.
